pages/analysis-step-versions/[id].js

```diff
@@ -124,7 +124,9 @@ export async function getServerSideProps({ params, req, query }) {
     return {
       props: {
         analysisStepVersion,
-        pageContext: { title: analysisStepVersion.uuid },
+        pageContext: {
+          title: `${analysisStepVersion.analysis_step.title} ${"Version"}`,
```

Comment:
Since “Version” is just a static string, you don’t need it in a `${}`. Just `${analysisStepVersion.analysis_step.title} Version`

Reply:
ok